.github/workflows/tests.yml

```diff
@@ -85,15 +84,15 @@ jobs:
           mkdir -p /usr/local/miniconda/envs
           sudo cp -r ~/osx-conda /usr/local/miniconda/envs/anaconda-client-env
 
-      # retaining this for the next devel cycle
-      # - name: Build SLiM
-      #   run: |
-      #     git clone https://github.com/messerlab/SLiM.git
-      #     cd SLiM
-      #     mkdir -p Release
-      #     cd Release
-      #     cmake -D CMAKE_BUILD_TYPE=Release ..
-      #     make -j 2
+      - name: Build SLiM
+        if: steps.cache.outputs.cache-hit != 'true' &&
+          (matrix.os == 'macos-latest' || matrix.os == 'ubuntu-latest')
+        run: |
+          git clone https://github.com/messerlab/SLiM.git
+          mkdir -p SLiM/Release
+          cd SLiM/Release
+          cmake -D CMAKE_BUILD_TYPE=Release ..
+          make -j 2
```

Comment:
what's up with these changes?

Reply:
just a bit of a noisy diff it looks like from uncommenting that code block so we build on macos / ubuntu, but guess we don't need it?

Reply:
commenting these out